bluebrain/deployment/environments/externals.yaml

```diff
@@ -63,10 +63,10 @@ spack:
     - llvm@13.0.0
     - llvm@12.0.1
     - llvm@11.1.0
-    - nvhpc@21.2 install_type=network
     - nvhpc@21.11
     - nvhpc@22.1
     - nvhpc@22.2
```

Comment:
should remove any older version?

Reply:
may be 22.1 can be also removed if we want to keep last stable 22.2 and new 22.3? (all older versions are in archive anyway if we want to grab them for testing/debugging)

Reply:
OK, I can drop 22.1 and CUDA 11.5.1 that went with it.
Also wondering if we need CUDA 11.0.2 for any reason any more.

Reply:
Dropped 22.1 in 64c794f04e.